packages/holocron/README.md

```diff
@@ -14,6 +14,8 @@
 
 ## ✨ Features
 
+* Adds an ErrorBoundary around every Module to prevent a single Module from taking down a page once an error occurs in the module
+
```

Comment:
I know this effort was about error boundaries but it will look odd when merged to master showing Holocron having only one feature.

Reply:
@mtomcal,

I agree. If you and the team could give some more features to add here I can. I believe the `ErrorBoundary` should be under Features and not under another part of the documentation.